### DISCUSSION TITLE
<title> How to submit my contact form in github.io

### DISCUSSION THREAD

**Discussion by BoudinAhmed** [community/community#103258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have spent days trying to find a solution. I went from trying to use php (no smart) to using google sheet. Which google sheet was working in my local environment but when I published it, I am getting a 405 :( 
Any help would be great

**Comment by gil-son** [comment]
A 405 HTTP status code typically indicates that the HTTP method (GET, POST, etc.) used for the request is not allowed on the requested resource. This error can occur for various reasons:

- Verify the requirements to use API from Google sheet
- Check CORS Configuration
- Verify Request Method (GET, POST, PUT, DELETE, etc)

**Reply by BoudinAhmed** [reply]
I'll do more research and get back if I find a solution

**Reply by BoudinAhmed** [reply]
Resolved it by removing the async function in my code. Able to post!

**Reply by gil-son** [reply]
Great job!
